### PR TITLE
Fix PEFT breaking change

### PR DESCRIPTION
**Pull request by younesbelkada**
cc @pacman100 @sayakpaul

### CODE REVIEW

**Review by sayakpaul — APPROVED** [review]
Thanks for the quick resolve. Would it make sense to do a patch release for this?

**Review by pacman100 — COMMENTED** [review]
Hello Younes, thank you for the fix but is there a failure case because i think `vocab_size` would only be available for transformers model.

**Review by pacman100 — APPROVED** [review]
Just saw the issue offline, Thank you for the quick fix! 🚀

File: src/peft/utils/save_and_load.py [hunk 1]
```diff
@@ -136,8 +136,18 @@ def get_peft_model_state_dict(
     elif save_embedding_layers == "auto":
         vocab_size = getattr(getattr(model, "config", None), "vocab_size", None)
         model_id = getattr(config, "base_model_name_or_path", None)
+
+        # For some models e.g. diffusers the text config file is stored in a subfolder
+        # we need to make sure we can donwload that config.
+        has_remote_config = model_id is not None and file_exists(model_id, "config.jon")
```
Comment (pacman100) [inline src/peft/utils/save_and_load.py:142]:
```suggestion
        has_remote_config = model_id is not None and file_exists(model_id, "config.json")
```

**Review by pacman100 — APPROVED** [review]
Thank you Younes for the fix! ✨ left a suggestion

File: src/peft/utils/save_and_load.py [hunk 2]
```diff
@@ -136,8 +136,18 @@ def get_peft_model_state_dict(
     elif save_embedding_layers == "auto":
         vocab_size = getattr(getattr(model, "config", None), "vocab_size", None)
         model_id = getattr(config, "base_model_name_or_path", None)
+
+        # For some models e.g. diffusers the text config file is stored in a subfolder
+        # we need to make sure we can donwload that config.
```
Comment (pacman100) [inline src/peft/utils/save_and_load.py:141]:
```suggestion
        # we need to make sure we can download that config.
```

**Pull request closed without merging**